<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMotorBoundsDialog</class>
 <widget class="QDialog" name="EditMotorBoundsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="formGroupBox">
     <property name="title">
      <string>Motor Bounds:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>B (Tilt):</string>
        </property>
        <property name="buddy">
         <cstring>minLineEditB</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="minLineEditB"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="hereMinButtonB">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>X:</string>
        </property>
        <property name="buddy">
         <cstring>minLineEditX</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="minLineEditX"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="hereMinButtonX">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Y:</string>
        </property>
        <property name="buddy">
         <cstring>minLineEditY</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="minLineEditY"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="hereMinButtonY">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Z (Zoom):</string>
        </property>
        <property name="buddy">
         <cstring>minLineEditZ</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="minLineEditZ"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="hereMinButtonZ">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>A (Spin):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lower Bound</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="maxLineEditA"/>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="hereMaxButtonA">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Upper Bound</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="maxLineEditB"/>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="maxLineEditX"/>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="maxLineEditY"/>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="maxLineEditZ"/>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="hereMaxButtonB">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="hereMaxButtonX">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="hereMaxButtonY">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="hereMaxButtonZ">
        <property name="text">
         <string>Here</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditMotorBoundsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditMotorBoundsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
